### DISCUSSION TITLE
Alpha 146

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4177] [Release Notes]
- Fixed compatibility with some older plugins
- Profiles now sorted by group first
- Fixed toggle switch and color scheme preview appearance in high-contrast mode

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.146'>Alpha 146</a>.</em>

**Comment by zhuzhucui** [comment]
此版本lrzsz不生效

**Comment by zhuzhucui** [comment]
winscp热键不生效
